data/Templates/Ccda/ValueSet/_AllergyCategory.liquid

```diff
@@ -16,7 +16,7 @@
     {% when "419199007" -%}
         "other",
     {% else -%}
-        "{{ code }}",
+        "",
```

Comment:
Why not leave original value?

Reply:
Per the offline discussion, an empty string will be used for the unmapped cases in this pr. And creating `extensions` to handle it later.